gpt_engineer/preprompts/generate

```diff
@@ -1,8 +1,5 @@
-You will get instructions for code to write.
-You will write a very long answer. Make sure that every detail of the architecture is, in the end, implemented as code.
```

Comment:
Would be great to run benchmark after deleting this line!

Are you confident that it will not deteriorate the output?

(gpt4 can sometimes stop before its done)

Reply:
Oh, I didn't delete these lines, I moved them into the preprompts/roadmap file so a near-identical use_qa file won't be necessary any longer